Portal/src/Datahub.Infrastructure/Services/Helpers/HealthCheckHelper.cs

```diff
@@ -607,53 +607,63 @@ public async Task<IntermediateHealthCheckResult> CheckWebApp(InfrastructureHealt
             var errors = new List<string>();
             var status = InfrastructureHealthStatus.Healthy;
 
-            await using var ctx = await dbContextFactory.CreateDbContextAsync();
```

Comment:
Github's diff thinks this is a bigger change than it is, but it is just wrapping the thing in a try/catch